aldryn_forms/boilerplates/legacy/templates/aldryn_forms/base_form.html

```diff
@@ -0,0 +1,18 @@
+{# This is a very nasty hack to redirect the user #}
+{% if form_success_url %}
+<script type="text/javascript">
+    window.location.replace("{{ form_success_url }}");
```

Comment:
that is correct, but apparently not possible due to some response handle issues from the backend (cms related) /cc @czpython